echidna/core/fill_spectrum.py

```diff
@@ -1,3 +1,4 @@
+import root_help
```

Comment:
I'm slightly confused why you've imported it here, as this script does not take command line arguments, so no `--help` option anyway.